src/main/java/count/Count.java

```diff
@@ -0,0 +1,85 @@
+package count;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+import count.action.Action;
+import count.action.Deactivate;
+import count.exception.CountException;
+import count.exception.IncorrectFormatException;
+
+/**
+ * Count is a lightweight helper with an inbuilt to-do list
+ * @author Kieran Koh Jun Wei
+ */
+public class Count {
+    private TaskList ls;
```

Comment:
I second the other reviewer's comment that ls is not descriptive of what this does. Perhaps you could call it `taskList`